woocommerce/woocommerce
# [Experimental] Product Filters Redesign > Clear Button: Fix CSS Not Found error

### Changes proposed in this Pull Request:

This PR fixes an error in the console due to the lack of the CSS file for the Product Filter Clear Button block.





Closes #47480.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Open the console in your browser;
2. Visit the post/site editor;
3. Make sure you don't see any error related to CSS not being found in the page.
  3.1. The error that was appearing was`/wp-content/plugins/woocommerce/assets/client/blocks/product-filter-clear-button.css?ver=wc-9.0.0 net::ERR_ABORTED 404 (Not Found)`.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 
Fix CSS error on the Product Filter Clear Button block

</details>
